cli/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "zip-cli"
+version = "0.0.0"
```

Comment:
Change this to `0.0.1` for alpha or `0.1.0` for beta.

Reply:
Done!